### ISSUE TITLE
Extension Error: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by vkaracic** [GoogleChrome/lighthouse#1200]
**Lighthouse Version**: 1.3.0
**Chrome Version**: 54.0.2840.100
**Error Message**: Failed to construct 'URL': Invalid URL
**Stack Trace**:
 ```
TypeError: Failed to construct 'URL': Invalid URL
    at artifacts.AnchorsWithNoRelNoopener.usages.filter.anchor (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:689:25)
    at Array.filter (native)
    at Function.audit (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:689:8)
    at Promise.resolve.then._ (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11365:20)
```

This happened when testing out the chrome plugin on http://developers.google.com

**Comment by wardpeet** [comment]
@vkaracic do you have any chrome plugins installed? Do you have any options deselected on the extension?

// dupe of #1194

**Comment by wardpeet** [comment]
Thanks for the feedback, error should be fixed in #1218

**Comment by nayyarsameer** [comment]
First you need to remove ARC from your system.
After removing ARC just run the following command using terminal
<pre>
$ sudo apt-get purge google-chrome-stable
$ rm ~/.config/google-chrome/ -rf
</pre>
the first command removes the package while the second command will remove custom settings such as plugins, etc.
also run the following command in terminal
<pre>
$ sudo apt-get -f autoremov
</pre>

Now just make sure chrome completely remove or not.
go to terminal and run the following command
<pre>
$ google-chrome --version
</pre>
This should not give you the version of chrome


now you need to reinstall latest chrome version 
google-chrome-stable is available on a 3rd Party Repository: Google Chrome (for Stable).

Follow the instruction for installation:

Add Key:

wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
Set repository:

echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | sudo tee /etc/apt/sources.list.d/google-chrome.list
Install package:

sudo apt-get update 
sudo apt-get install google-chrome-stable


now check the version of your chrome again
<pre>
$ google-chrome --version
</pre>

Now you just intsall ARC from google store

please write comment if found any bugs
